packages/contract-wrappers/src/contract_wrappers/forwarder_wrapper.ts

```diff
@@ -40,27 +41,28 @@ export class ForwarderWrapper extends ContractWrapper {
      * 5% of ETH value is reserved for paying fees to order feeRecipients (in ZRX) and forwarding contract feeRecipient (in ETH).
      * Any ETH not spent will be refunded to sender.
-     * @param   signedOrders         An array of objects that conform to the SignedOrder interface. All orders must specify the same makerAsset.
-     *                               All orders must specify WETH as the takerAsset
-     * @param   takerAddress         The user Ethereum address who would like to fill this order. Must be available via the supplied
-     *                               Provider provided at instantiation.
-     * @param   ethAmount            The amount of eth to send with the transaction (in wei).
-     * @param   signedFeeOrders      An array of objects that conform to the SignedOrder interface. All orders must specify ZRX as makerAsset and WETH as takerAsset.
-     *                               Used to purchase ZRX for primary order fees.
-     * @param   feePercentage        The percentage of WETH sold that will payed as fee to forwarding contract feeRecipient.
-     *                               Defaults to 0.
-     * @param   feeRecipientAddress  The address that will receive ETH when signedFeeOrders are filled.
-     * @param   txOpts               Transaction parameters.
+     * @param   signedOrders            An array of objects that conform to the SignedOrder interface. All orders must specify the same makerAsset.
+     *                                  All orders must specify WETH as the takerAsset
+     * @param   takerAddress            The user Ethereum address who would like to fill this order. Must be available via the supplied
+     *                                  Provider provided at instantiation.
+     * @param   ethAmount               The amount of eth to send with the transaction (in wei).
+     * @param   signedFeeOrders         An array of objects that conform to the SignedOrder interface. All orders must specify ZRX as makerAsset and WETH as takerAsset.
+     *                                  Used to purchase ZRX for primary order fees.
+     * @param   feePercentage           The percentage of WETH sold that will payed as fee to forwarding contract feeRecipient.
+     *                                  Defaults to 0.
+     * @param   feeRecipientAddress     The address that will receive ETH when signedFeeOrders are filled.
+     * @param   orderTransactionOpts    Transaction parameters.
      * @return  Transaction hash.
      */
+    @decorators.asyncZeroExErrorHandler
     public async marketSellOrdersWithEthAsync(
         signedOrders: SignedOrder[],
         takerAddress: string,
         ethAmount: BigNumber,
         signedFeeOrders: SignedOrder[] = [],
         feePercentage: number = 0,
         feeRecipientAddress: string = constants.NULL_ADDRESS,
-        txOpts: TransactionOpts = {},
+        orderTransactionOpts: OrderTransactionOpts = {},
```

Comment:
We default this to { shouldValidate: true } in exchange_wrapper and probably should do the same here